src/python/library/tritonclient/grpc/__init__.py

```diff
@@ -26,7 +26,17 @@
 
 try:
     import grpc
-    from google.protobuf.json_format import MessageToJson
+    from tritonclient.grpc import model_config_pb2
```

Comment:
I was trying to be safe here so that if any of the dependencies required was not installed we can catch it and return proper error message. The user may have installed grpc but has not specified `tritonclient[grpc/all]` so it would help us catch those issues.